libraries/botbuilder-lg/tsconfig.json

```diff
@@ -4,9 +4,10 @@
     "lib": ["es2015", "dom"],
     "outDir": "lib",
     "rootDir": "src",
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "esModuleInterop": true,
```

Comment:
Adding this without vetting existing imports can cause problems.

```
~/src/github/joshgummersall/tsplay > cat src/moment.ts
───────┬───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
       │ File: src/moment.ts
───────┼───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
   1   │ import moment from "moment";
   2   │ import * as momentStar from "moment";
   3   │
   4   │ console.log({ moment, momentStar });
   5   │
   6   │ moment();
   7   │ momentStar();
───────┴───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
~/src/github/joshgummersall/tsplay > yarn ts-node src/moment.ts
yarn run v1.22.10
$ ts-node src/moment.ts

/Users/jpg/src/github/joshgummersall/tsplay/node_modules/ts-node/src/index.ts:513
    return new TSError(diagnosticText, diagnosticCodes)
           ^
TSError: ⨯ Unable to compile TypeScript:
src/moment.ts:7:1 - error TS2349: This expression is not callable.
  Type 'typeof moment' has no call signatures.

7 momentStar();
  ~~~~~~~~~~

  src/moment.ts:2:1
    2 import * as momentStar from "moment";
      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Type originates at this import. A namespace-style import cannot be called or constructed, and will cause a failure at runtime. Consider using a default import or import require here instead.

    at createTSError (/Users/jpg/src/github/joshgummersall/tsplay/node_modules/ts-node/src/index.ts:513:12)
    at reportTSError (/Users/jpg/src/github/joshgummersall/tsplay/node_modules/ts-node/src/index.ts:517:19)
    at getOutput (/Users/jpg/src/github/joshgummersall/tsplay/node_modules/ts-node/src/index.ts:752:36)
    at Object.compile (/Users/jpg/src/github/joshgummersall/tsplay/node_modules/ts-node/src/index.ts:968:32)
    at Module.m._compile (/Users/jpg/src/github/joshgummersall/tsplay/node_modules/ts-node/src/index.ts:1056:42)
    at Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Object.require.extensions.<computed> [as .ts] (/Users/jpg/src/github/joshgummersall/tsplay/node_modules/ts-node/src/index.ts:1059:12)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Unless we have really good test coverage I would prefer to remove `esModuleInterop` and just use the `import *` syntax until we have a chance to fully resolve the potential implications.